run_test.sh

```diff
@@ -35,6 +35,14 @@ case $TEST in
      $HOME/ci/build_ports.sh
      rc=$?
      ;;
+  "BUILD_TARGETS_WINDOWS")
+     $HOME/ci/test_build_targets.sh
```

Comment:
I think you can remove this case and add it together with the one above using:

`"BUILD_TARGETS" | "BUILD_TARGETS_WINDOWS")`